fastlane/Fastfile

```diff
@@ -585,6 +585,31 @@ platform :ios do
     )
   end
 
+  # Upload the localized metadata (from `fastlane/metadata/`) to App Store Connect
+  #
+  # @option [Boolean] with_screenshots (default: false) If true, will also upload the latest screenshot files to ASC
+  #
+  desc 'Upload the localized metadata to App Store Connect, optionally including screenshots.'
+  lane :update_metadata_on_app_store_connect do |options|
+    # Skip screenshots by default. The naming is "with" to make it clear that
+    # callers need to opt-in to adding screenshots. The naming of the deliver
+    # (upload_to_app_store) parameter, on the other hand, uses the skip verb.
+    with_screenshots = options.fetch(:with_screenshots, false)
+    skip_screenshots = !with_screenshots
+
+    upload_to_app_store(
+      app_identifier: APP_STORE_VERSION_BUNDLE_IDENTIFIER,
```

Comment:
I didn't set the `team_id` which was being set in the `Deliverfile`, as I believe it isn't always necessary. I've seen some other lanes using `get_required_env('EXT_EXPORT_TEAM_ID')` but I wasn't sure whether it was safe to use it in this case, as the context in which this lane is run might be different.

Anyway, please let me know if `team_id` is in fact needed and I'll amend the PR.

Reply:
> This means that some (if not most) of engineers in Apps Infra, who will at some point rotate in doing the release management for different apps, or cover for someone else in the team while they are away, will end up having multiple App Store Connect team registered with their Apple ID, and thus it's still important to pass an explicit value here as letting fastlane guess it implicitly won't work on a multiple-team setup like ours.

Got it! Yes, I wasn't 100% sure about that (plus, though WPiOS uses the `team_id` for `testflight()`, it [doesn't use it](https://github.com/wordpress-mobile/WordPress-iOS/blob/trunk/fastlane/lanes/localization.rb#L382) for upload metadata 🤔). Thanks for clarifying!

And also, good to know the additional context and info on these vars and the secrets file, very helpful 👍